src/main/java/foxtrot/uniform/charlie/kilo/service/implementation/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,52 @@
+package foxtrot.uniform.charlie.kilo.service.implementation;
+
+import foxtrot.uniform.charlie.kilo.dao.ShoppingCartDao;
+import foxtrot.uniform.charlie.kilo.dao.TicketDao;
+import foxtrot.uniform.charlie.kilo.lib.Inject;
+import foxtrot.uniform.charlie.kilo.lib.Service;
+import foxtrot.uniform.charlie.kilo.model.MovieSession;
+import foxtrot.uniform.charlie.kilo.model.ShoppingCart;
+import foxtrot.uniform.charlie.kilo.model.Ticket;
+import foxtrot.uniform.charlie.kilo.model.User;
+import foxtrot.uniform.charlie.kilo.service.ShoppingCartService;
+import java.util.List;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+    @Inject
+    private TicketDao ticketDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(movieSession);
+        ticket.setUser(user);
+        ticketDao.add(ticket);
```

Comment:
if there is no shopping cart for the user - we still save ticket to the db, is it ok?

Reply:
User always has a shopping card, we create a shopping card when registering a new user